fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml

```diff
@@ -412,6 +415,27 @@ components:
         type: string
       example: "https://example.com/myservice?auth-id=sf3x;authResult=failure"
 
+    LoARetrievalInformation:
+      name: LoARetrievalInformation
+      required: true
+      in: header
+      schema:
+        type: string
+        enum:
+          - "fromTppWithAvailableConsent"
```

Comment:
FROM_TPP_WITH_AVAILABLE consent looks like more suitable name for constant value (same for other enum values)

See:
https://stackoverflow.com/questions/13233137/javascript-enum-pattern-naming-convention